Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:14-bullseye
+FROM node:20-bookworm
```

Comment:
we should also change the testsuite to target node 20 too now that I think about it

Reply:
that would require also changing the node version in package.json and the actual yarn build is still in buster, i believe that's a separate unrelated change

Reply:
if we're changing the runtime to node 20 we should be testing against node 20

Reply:
@sl0thentr0py why does adding 20 here not enough: https://github.com/getsentry/craft/blob/65c79d83a521cfe5f1c9f19a6ed4a46ccaa719f5/.github/workflows/build.yml#L14

Reply:
For testing I mean

Reply:
ah just for testing, makes sense, sorry was irl last night so distracted